tools/lambda-promtail/lambda-promtail/promtail.go

```diff
@@ -103,6 +116,18 @@ func (b *batch) createPushRequest() (*logproto.PushRequest, int) {
 	return &req, entriesCount
 }
 
+func (b *batch) flushBatch(ctx context.Context) error {
+	fmt.Println("flusing batch")
```

Comment:
nit: can we remove this println? I'm not against adding some debug logging, but that can be a followup PR and should use the log level conventions.

Reply:
commited it by mistake, will remove it